examples/plot_simulate_evoked_mpi.py

```diff
@@ -0,0 +1,43 @@
+"""
+===============
+Simulate dipole
+===============
+
+This example demonstrates how to run simulations with MPI using HNN-core.
+"""
+
+# Authors: Mainak Jas <mainak.jas@telecom-paristech.fr>
+#          Sam Neymotin <samnemo@gmail.com>
+
+import os.path as op
+
+###############################################################################
+# Let us import hnn_core
+
+import hnn_core
+from hnn_core import simulate_dipole, read_params, Network, MPIBackend
+
+hnn_core_root = op.join(op.dirname(hnn_core.__file__), '..')
+
+###############################################################################
+# Then we read the parameters file
+params_fname = op.join(hnn_core_root, 'param', 'default.json')
+params = read_params(params_fname)
+
+###############################################################################
+# Create the Network instance to use for simulations
+net = Network(params)
+
+###############################################################################
+# Now let's simulate the dipole using the MPI backend. This will
+# start the simulation across the number of processors (cores)
+# specified by n_procs using MPI. The 'mpiexec' launcher is for
+# openmpi, which must be installed on the system
+with MPIBackend(n_procs=2, mpi_cmd='mpiexec'):
```

Comment:
should we just update an existing example rather than create a new one? it's just one line change, right? Also, you can add your name to the list of authors!

Reply:
Removed extra example.

https://github.com/jonescompneurolab/hnn-core/pull/79/commits/65e39fd7262b58332677132a047b6c3b0387943c#diff-a3ce39929b27923f592792d215f8e6fdR9-R79